README.md

```diff
@@ -1,12 +1,19 @@
-# devdevdev
+![](https://img.shields.io/github/license/reeversedev/devdevdev)
 
```

Comment:
How about adding the name of the project as a highlight here?

Reply:
Great idea!